### DISCUSSION TITLE
new name of the organisation as deleted organisation

### DISCUSSION THREAD

**Discussion by duskohu** [community/community#34816] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,
I had an organization that I deleted,
I wanted to create the same organization after deleting it, but github won't let me.
How can do it?

Thx.

**Comment by jfullstackdev** [comment, marked as answer]
yup, because it will take time for it to be available,

there are always consequences not only in that but related to that, like retired repo names,

as far as I know, for accounts, I'm not very sure, is 90 days, but that will depend as there is the non-releasing usernames even if it seems not active 

posting it here might be noticed by the Staff members, other than that there is the Support but again there is no assurance

https://support.github.com/contact/account
